Sources/muterCore/fileOperations.swift

```diff
@@ -2,9 +2,12 @@ import SwiftSyntax
 import Foundation
 
 // MARK - Source Code
-func sourceCode(fromFileAt path: String) -> SourceFileSyntax? {
+
+func sourceCode(fromFileAt path: String) -> SourceCodeInfo? {
     let url = URL(fileURLWithPath: path)
-    return try? SyntaxTreeParser.parse(url)
+    return (try? SyntaxParser.parse(url))
+        .map { (code: $0, path: url.absoluteString) }
+        .map (SourceCodeInfo.init)
```

Comment:
```suggestion
        .map(SourceCodeInfo.init)
```

Maybe just personal style? I can see the appeal of the space so it matches the previous line, but I've never seen this style actually used before.